### PR TITLE
Implement a status bar and put hardware model name and a popup in there

### PR DESCRIPTION
**Pull request by andrewdavidmackenzie**
Part of #140 
This will make work on pi layout easier I think.

### CODE REVIEW

File: src/piggui.rs [hunk 1]
```diff
@@ -393,113 +480,37 @@ impl Application for Gpio {
...
-
-        let content = container(main_row)
+        let content = container(main_col)
             .height(Length::Fill)
             .width(Length::Fill)
             .align_x(iced::alignment::Horizontal::Center)
             .center_x()
             .center_y()
-            .padding(10);
+            .padding(padding);
```
Comment (sundaram123krishnan) [inline src/piggui.rs:513]: Instead u can pass an array, 
.padding([top, right, bottom, left])

File: src/piggui.rs [hunk 2]
```diff
@@ -393,113 +480,37 @@ impl Application for Gpio {
...
-                    .on_press(if !self.show_toast {
-                        // Add a new toast if `show_toast` is false
-                        Message::Load
-                    } else {
-                        // Close the existing toast if `show_toast` is true
-                        let index = self.toasts.len() - 1;
-                        Message::Toast(ToastMessage::Close(index))
-                    }),
-            );
+        let padding = Padding {
```
Comment (sundaram123krishnan) [inline src/piggui.rs:500]: Ig this is not necessary

**Review by sundaram123krishnan — COMMENTED** [review]
LGTM

File: src/piggui.rs (hunk 2 above)
Comment (andrewdavidmackenzie) [reply src/piggui.rs:507]: I did this so that status bar is closer to the bottom of the window, and no wasted space below it. "bottom" is 0.0

Comment (andrewdavidmackenzie) [reply src/piggui.rs:507]: I want to add a subtle line (Rule::) about it, maybe in a black color. But I didn't have time to see how to style it...

**Pull request merged**